pipe_vessels/postgres/tracks.py

```diff
@@ -1,16 +1,30 @@
 import sys
-from pipe_vessels.postgres.converter import convert_json_to_csv, normalize_geography, normalize_array
-
+import json
+import csv
 
 def record_to_row(record):
```

Comment:
Add some basic description of what this script does: `Loads the new line json of tracks and create csv so it can be imported by postgres.`